retroshare-gui/src/gui/common/GroupTreeWidget.cpp

```diff
@@ -167,8 +160,8 @@ void GroupTreeWidget::addToolButton(QToolButton *toolButton)
 
 	/* Initialize button */
 	toolButton->setAutoRaise(true);
-	toolButton->setIconSize(ui->displayButton->iconSize());
-	toolButton->setFocusPolicy(ui->displayButton->focusPolicy());
+	toolButton->setIconSize(QSize(24,24));

```

Comment:
Bad way to do it: you need 24*QFontMetricsF(font()).height()/14.0, so it remains proportional to font size and keeps the same look on high DPI screens.

Reply:
OK,
For next, could you test with this:
The Qt::AA_EnableHighDpiScaling application attribute, introduced in Qt 5.6, enables automatic scaling based on the monitor's pixel density.
https://doc.qt.io/qt-5/highdpi.html

Reply:
Now, I use:
	int i = qt_defaultDpi();
	auto desktopWidget = QApplication::desktop();
	auto y = desktopWidget->logicalDpiY();

	toolButton->setIconSize(QSize(24*y/i,24*y/i));
If it's OK, we can make it static and in utils function.